apps/ubuntu_bootstrap/lib/pages/storage/storage_model.dart

```diff
@@ -71,12 +72,18 @@ class StorageModel extends SafeChangeNotifier {
     this._telemetry,
     this._product,
     this._session,
-  );
+  ) : showAdvanced = false;
 
   final StorageService _storage;
   final TelemetryService? _telemetry;
   final ProductService _product;
   final SessionService _session;
+  bool showAdvanced;
```

Comment:
Nitpick: could we make that private to prevent accidentally changing it without calling `notifyListeners()`?
```suggestion
  var _showAdvanced = false;
  
  bool get showAdvanced => _showAdvanced;
```

Reply:
On second though: maybe this doesn't need to be part of the storage model at all? This is only needed for the UI state of the new page, right?

Reply:
yes actually, its just tracking whether or not the user has clicked to expand out to show the full list of advanced options.